src/peergos/server/tests/MultiUserTests.java

```diff
@@ -493,6 +495,37 @@ public void followPeergos() throws Exception {
         assertTrue("Receive a follow request", u1Requests.size() > 0);
     }
 
+    @Test
+    public void sendFeedbackToPeergos() throws Exception {
+        UserContext peergos = PeergosNetworkUtils.ensureSignedUp("peergos", "testpassword", network, crypto);
+        UserContext newUser = PeergosNetworkUtils.ensureSignedUp("w", "w", network, crypto);
```

Comment:
best to choose a random username (random() will do) to ensure independence from other tests (we set up one cluster for the whole test suite)

Reply:
That might be why it passes when run on it's own.

Reply:
Ah! OK cool. I'll make that change now.

Reply:
Hmmm, still failing. I've replicated the error locally so will work on that later tonight.